Makefile

```diff
@@ -87,3 +87,11 @@ show-data-model:
 	# With --dev, you'll see the currently experimental parts, as well.
 	# Use --help to learn more. 
 	./flexmeasures/data/scripts/visualize_data_model.py --uml --dev
+
+clean_db:
+	sudo -i -u postgres dropdb -U postgres --if-exists ${db_name}
```

Comment:
I was wondering whether all of these commands would also work without `sudo -i -u postgres` prepended? (That seems to be the case on my local dev environment, but maybe not for everyone.)

Reply:
i have tried it without this, but it fails.